recipes/enum-flags/0.1a/conanfile.py

```diff
@@ -0,0 +1,34 @@
+from conans import ConanFile, tools
+import os
+
+
+class EnumFlagsConan(ConanFile):
+    name = "enum-flags"
+    description = "Bit flags for C++11 scoped enums"
+    homepage = "https://github.com/grisumbras/enum-flags"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("bitmask", "enum")
+    license = "MIT"
+    options = {"forbid_implicit_conversions": [True, False]}
+    default_options = {"forbid_implicit_conversions": True}
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake"
    no_copy_source = True
```
https://docs.conan.io/en/latest/reference/conanfile/attributes.html#no-copy-source